lib/utils.sh

```diff
@@ -478,7 +478,11 @@ get_debian_major_version() {
     DEB_MAJ_VER=""
     does_file_exist /etc/debian_version
     if [ "$FNRET" = 0 ]; then
-        DEB_MAJ_VER=$(cut -d '.' -f1 /etc/debian_version)
+        if grep -q "sid" /etc/debian_version; then
```

Comment:
Blocker : this is only true at the moment, sid will never move, in three years the if would have to be changed by debian12, you must find another logic (Maybe use a keyword like testing experimental, and add a case test to update?)